### PR TITLE
[api.keyman.com] Lexical model schema drafts

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

**Comment by eddieantonio** [comment]
I'm assuming some of the string patterns in `model_info.source.json` will be relaxed in the future. Other than that, LGTM!

File: schemas/package.json [hunk 1]
```diff
@@ -239,6 +242,42 @@
...
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "rtl": {
```
Comment (eddieantonio) [inline schemas/package.json]: Should this be `isRTL` to be consistent with `#/definitions/ModelInfo`?

Comment (mcdurdin) [reply schemas/package.json]: Consistency would be a nice thing to have. Sadly I've chosen to use `rtl` to be consistent with `package.json#/definitions/keyboards/rtl`... so at least the fields are relatively consistent within a single file.

**Comment by mcdurdin** [comment]
> I'm assuming some of the string patterns in `model_info.source.json` will be relaxed in the future.

I'm not sure which string patterns you mean? If you mean the filename patterns, I don't plan to relax them because they are enforcing a structure in the repository and distribution. There's no requirement for models distributed outside our repo/Keyman Cloud distribution model to follow those patterns, but then they don't need a .model_info file either. Hope that makes sense?

Note: I realised I annotated the 11.0 version of the kmp.json metadata file. Will correct to annotating 12.0 version (requires a separate PR to seed the 12.0 docs).

**Comment by mcdurdin** [comment]
Not merging yet: need to sort out versioning on (all) schema files

**Pull request merged**